enhancements/upgrade-bundle.md

```diff
@@ -0,0 +1,138 @@
+---
+title: upgrade-bundle
+authors:
+  - "@rashmigottipati"
+reviewers:
+  - "@joelanford"
+  - "@estroz"
+  - "@shawn-hurley"
+  - "@jmrodri"
+approvers:
+  - TBD
+creation-date: 2020-06-11
+last-updated: 2020-06-11
+
+---
+
+# Upgrade Bundle
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions
+
+1. How do we handle multiple subscriptions? 
+2. Should we explicitly pass new version number as a flag or can we get it from bundle image?
+    * Based on discussions, can extract the version number from bundle image. 
+3. How do we verify that the upgrade is successful 
```

Comment:
For now let's just check if the CSV has status `Installed` in the correct set of namespaces. Eventually OLM will/should surface the result of a deeper inspection of a successful upgrade (ex. pod readiness) somewhere, like in a CSV's status perhaps, but that's clearly out of scope for this EP.

Reply:
Thanks for throwing some light on these open questions 👍  , I've incorporated your feedback into those.